src/test/java/org/gridsuite/modification/server/modifications/LineAttachToVoltageLevelWithNewVoltageLevelTest.java

```diff
@@ -0,0 +1,115 @@
+/*
+  Copyright (c) 2022, RTE (http://www.rte-france.com)
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.modifications;
+
+import org.gridsuite.modification.server.ModificationType;
+import org.gridsuite.modification.server.dto.*;
+import org.gridsuite.modification.server.utils.MatcherModificationInfos;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.gridsuite.modification.server.utils.MatcherLineAttachToVoltageLevelInfos.createMatcherLineAttachToVoltageLevelInfos;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+/**
+ * @author David Braquart <david.braquart at rte-france.com>
+ */
+public class LineAttachToVoltageLevelWithNewVoltageLevelTest extends AbstractNetworkModificationTest {
```

Comment:
Rename by LineAttachToNewVoltageLevelTest ?

Reply:
I think we should keep <class>Test / <class>With<specialcase>Test.
it's easier to see we are testing the same class.
and it is consistent with LineAttachToVoltageLevelTest / LineAttachToVoltageLevelWithNewVoltageLevelTest